.idea/modules.xml

```diff
@@ -2,7 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://C:\AndroidStudioProject\MoneySplitter\mobile-money-splitter-android\MoneySplitter.iml" filepath="C:\AndroidStudioProject\MoneySplitter\mobile-money-splitter-android\MoneySplitter.iml" />
```

Comment:
Maybe use `$PROJECT_DIR$` for this?

Reply:
This is autogenerated file and it should be ignored, but something went wrong...

Reply:
@IstrajI soryan, bolshe tak ne budu